skipruntime-ts/core/test/runtime.spec.ts

```diff
@@ -13,10 +13,30 @@ import type {
   Resource,
   Entry,
   ExternalSupplier,
+  SkipRuntime,
 } from "../src/skip-runtime.js";
 import { Sum, ValueMapper, initService } from "../src/skip-runtime.js";
 import { TimeCollection, ExternalService } from "../src/skipruntime_helpers.js";
 
+function getAll<K extends TJSON, V extends TJSON>(
+  runtime: SkipRuntime,
+  resource: string,
+  params: Record<string, string> = {},
+  reactiveAuth?: Uint8Array,
+): Entry<K, V>[] {
+  return runtime.getAll<K, V>(resource, params, reactiveAuth).values.values;
+}
+
+function getOne<V extends TJSON>(
+  runtime: SkipRuntime,
+  resource: string,
+  params: Record<string, string>,
+  key: string | number,
+  reactiveAuth?: Uint8Array,
+): V[] {
+  return runtime.getOne<V>(resource, params, key, reactiveAuth).values;
+}
+
```

Comment:
I'd prefer to drop these helper functions -- people may look at these as examples, and it'd be better to just use the actual API throughout

Reply:
The tests essentially use internal mechanics, so if people can look at this as an example he'll certainly have to rewrite them to avoid misuses.